specs/model-avro.spec/src/main/scripts/io/aklivity/zilla/specs/model/avro/config/event.yaml

```diff
@@ -23,7 +23,8 @@ telemetry:
         events:
           - qname: test.net0
             id: model.avro.validation.failed
-            message: VALIDATION_FAILED java.io.EOFException
+            name: MODEL_AVRO_VALIDATION_FAILED
+            message: java.io.EOFException
```

Comment:
We'll need to clean this up as well, to eliminate the exposure of Java internals.

Reply:
Note: This is a valid exception `message` returned from the `AvroRuntimeException` class.